sdk/spring/spring-cloud-azure-core/src/main/java/com/azure/spring/core/AzureSpringIdentifier.java

```diff
@@ -42,7 +43,12 @@ private AzureSpringIdentifier() {
     public static final String AZURE_SPRING_APP_CONFIG = "az-sp-cfg/" + VERSION;
     public static final String AZURE_SPRING_EVENT_HUBS = "az-sp-eh/" + VERSION;
     public static final String AZURE_SPRING_KEY_VAULT_SECRETS = "az-sp-kv/" + VERSION;
-    public static final String AZURE_SPRING_KEY_VAULT_CERTIFICATES = "az-sp-kv-ct/" + VERSION;
+
+    /**
+     * Azure Spring KeyVault Certificate Client.
+     * Not use "az-sp-kv-ct", because "az-sp-kv-ct" already used for JCA module in azure-security-keyvault-jca-user-agent-value-prefix.txt
+     */
+    public static final String AZURE_SPRING_KEY_VAULT_CERTIFICATES = "az-sp-kvc/" + VERSION;
```

Comment:
Here is a potential problem: `az-sp-kv-ct/3.11.0.beta.1` length is 25, which is larger is 24, it will cause problem.